forms-flow-bpm/forms-flow-bpm-camunda/src/main/resources/META-INF/resources/webjars/camunda/app/cockpit/styles/user-styles.css

```diff
@@ -74,4 +74,8 @@ aside li.active>a {
   font-size: 30px;
   padding: 7px;
   line-height: 36px;
-}
\ No newline at end of file
+}
+
+.ce-eol-banner {
```

Comment:
This change is to hide the red notice from Camunda on end of life of v7 by Oct 2025